spec/rails_helper.rb

```diff
@@ -6,7 +6,7 @@
 
 require "rspec/rails"
 require "shoulda/matchers"
-require "capybara/poltergeist"
+require 'selenium/webdriver'
```

Comment:
```suggestion
require "selenium/webdriver"
```

Reply:
Ah thanks for these changes! I actually didn't have them originally but these were changes houndbot wanted so I thought that was the way to go. Thanks for these!